diff --git a/node_modules/jssip/lib-es5/RTCSession.js b/node_modules/jssip/lib-es5/RTCSession.js
index 702e700..d4d614d 100644
--- a/node_modules/jssip/lib-es5/RTCSession.js
+++ b/node_modules/jssip/lib-es5/RTCSession.js
@@ -577,9 +577,9 @@ module.exports = /*#__PURE__*/function (_EventEmitter) {
       } // Don't ask for audio if the incoming offer has no audio section.
 
 
-      if (!mediaStream && !peerHasAudioLine && !rtcOfferConstraints.offerToReceiveAudio) {
-        mediaConstraints.audio = false;
-      } // Don't ask for video if the incoming offer has no video section.
+      // if (!mediaStream && !peerHasAudioLine && !rtcOfferConstraints.offerToReceiveAudio) {
+      //   mediaConstraints.audio = false;
+      // } // Don't ask for video if the incoming offer has no video section.
 
 
       if (!mediaStream && !peerHasVideoLine && !rtcOfferConstraints.offerToReceiveVideo) {
diff --git a/node_modules/jssip/lib/RTCSession.js b/node_modules/jssip/lib/RTCSession.js
index df9e50a..968e7e0 100644
--- a/node_modules/jssip/lib/RTCSession.js
+++ b/node_modules/jssip/lib/RTCSession.js
@@ -635,10 +635,10 @@ module.exports = class RTCSession extends EventEmitter
     }
 
     // Don't ask for audio if the incoming offer has no audio section.
-    if (!mediaStream && !peerHasAudioLine && !rtcOfferConstraints.offerToReceiveAudio)
-    {
-      mediaConstraints.audio = false;
-    }
+    // if (!mediaStream && !peerHasAudioLine && !rtcOfferConstraints.offerToReceiveAudio)
+    // {
+    //   mediaConstraints.audio = false;
+    // }
 
     // Don't ask for video if the incoming offer has no video section.
     if (!mediaStream && !peerHasVideoLine && !rtcOfferConstraints.offerToReceiveVideo)
